apps/examples/wc-next10/ButtonLogic.js

```diff
@@ -0,0 +1,68 @@
+import React, { useState, useRef } from 'react';
+import { createComponent } from '@lit-labs/react';
+import { PieButton, BUTTON_SIZE, BUTTON_VARIANT } from '@justeattakeaway/pie-button';
+
+const PieBtn = createComponent({
+    tagName: 'pie-button',
+    elementClass: PieButton,
+    react: React,
+    events: { onCustomEvent: 'CustomEvent' },
+});
+
+export default function PButton () {
+    const variantIndex = useRef(0)
+    const [count, setCount] = useState(0);
+    const [variantName, setVariantName] = useState(BUTTON_VARIANT.PRIMARY);
+
+    const increment = () => setCount(count + 1);
+    const decrement = () => setCount(count - 1);
```

Comment:
in React, it's quite common to prefix callbacks with `on` to indicate a function will be used as a callback handler for a click or key up events.

```suggestion
    const onIncrement = () => setCount(count + 1);
    const onDecrement = () => setCount(count - 1);
```